chainercv/datasets/mixup_soft_label_dataset.py

```diff
@@ -55,16 +60,21 @@ class MixUpSoftLabelDataset(dataset_mixin.DatasetMixin):
     .. [#mixup_1] Same as :obj:`dataset`.
     """
 
-    def __init__(self, dataset, n_class):
+    def __init__(self, dataset, n_class, alpha=1.0):
+        if alpha <= 0.0:
```

Comment:
note: NumPy documentation says `alpha` is non-negative but it rejects alpha of 0 raising Value Error of `a <= 0`.